enduro.toml

```diff
@@ -72,3 +72,6 @@ disabled = true
 [[hooks."prod"]]
 receiptPath = "./hack/production-system-interface"
 disabled = true
+
+[metadata]
+processNameMetadata = true
```

Comment:
This file is now missing the final newline.

In VSCode: `"files.insertFinalNewline": true` does it automatically when you save.